ui/js/components/perf/compare.js

```diff
@@ -108,3 +108,33 @@ treeherder.component('compareError', {
         newRevision: '<'
     }
 });
+
+treeherder.component('distributionGraph', {
+    template:`
+        <table class = "tool-tip-table"> 
+            <tr> 
+                <td class="value-column">{{$ctrl.minValue|abbreviatedNumber}}</td> 
+                <td class="distribution-column"><canvas id="distribution-graph-new" width="250" height="30"></canvas></td>  
+                <td class="value-column">{{$ctrl.maxValue|abbreviatedNumber}}</td> 
+            </tr>
+        </table>`,
+    bindings: {
+        replicates: '<'
+    },
+    controller: [ function () {
+        let ctrl = this;
+        let cvs = document.getElementById("distribution-graph-new");
+        let ctx = cvs.getContext("2d");
+        cvs.setAttribute("id", "distribution-graph-current");
+        ctrl.maxValue = Math.ceil(Math.max.apply(null, ctrl.replicates)*1.01);
+        ctrl.minValue = Math.floor(Math.min.apply(null, ctrl.replicates)/1.01);
```

Comment:
Playing with this a bit more, I wonder if we shouldn't change the logic here a bit in the cases where the difference between min/max is less than one, as it's hard to visualize the distribution in those cases:

<img width="415" alt="screen shot 2017-06-12 at 11 30 18 am" src="https://user-images.githubusercontent.com/20569/27041699-afcf2b7e-4f62-11e7-8693-195d794c8b54.png">

For the case, maybe just take raw min/max?